pkg/webhook/webhook.go

```diff
@@ -172,9 +190,39 @@ func (w *Webhook) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 			return
 		}
 		for _, gitjob := range gitjobs {
+			if gitjob.Spec.Git.Revision != "" {
+				continue
```

Comment:
Don't we want to update on a revision change?

Reply:
No, if revision is set we basically don't process this gitjob, as this gitjob is attached to a specific commit. There should be no watch/webhook logic applied to this gitjob. It is like user setting a fix commit for gitjob/gitRepo to point to.